components/director/pkg/graphql/schema.graphql

```diff
@@ -1241,6 +1242,7 @@ type FormationTemplate {
 	runtimeArtifactKind: ArtifactType
 	webhooks: [Webhook!] @sanitize(path: "graphql.field.formation_template.webhooks")
 	leadingProductIDs: [String!]
+	formationConstraints: [FormationConstraint!]!
```

Comment:
Should be formationConstraints: [FormationConstraint!] as the constraints are not mandatory for the template